README.md

```diff
@@ -12,23 +12,22 @@ This is a sample repository showcasing how to integrate the Native Checkout expe
     - [Application Creation](#application-creation)
     - [NVM](#nvm)
     - [Docker](#docker)
-    - [Carthage](#carthage)
 3. [Setup](#setup)
-4. [Additional Info](#additional-info)
-5. [Troubleshooting](#troubleshooting)
+    - [Set Client ID / Client Secret / Return URL](#1-set-client-id--client-secret--return-url)
+    - [Setup Dependencies](#2-setup-dependencies)
+4. [Troubleshooting](#troubleshooting)
```

Comment:
Should we add a reference section in here to link to the public repo/docs as well? I think we should have something that ties this back to our public SDK repo, but the docs part may be covered in other seconds sufficiently. Thoughts?

Reply:
This makes 100% sense, I'll add that!!!!!! :smile: